### ISSUE TITLE
kubeconfig files in /tmp group and world readable and not being deleted

### ISSUE THREAD

**Issue by ribx** [lensapp/lens#655]
**Describe the bug**
If you add a cluster, lens stores the kubeconfig file and places it in /tmp to be able to set it as env variable. Problem is, these files are group and world readable and could include authentication certs. The should also be deleted on exit.

**To Reproduce**
1. add a cluster
2. check /tmp for "*-kubeconfig" files

**Expected behavior**
These files should not be world or group readable and be deleted on exit.

**Screenshots**
```
% ls -l /tmp/*-kubeconfig
-rw-r--r-- 1 ribx ribx 2508 2020-08-07 09:23 /tmp/0sf12nq7u3j-9s16ssgcnys-kubeconfig
-rw-r--r-- 1 ribx ribx 2508 2020-08-06 14:01 /tmp/1a7gohnoe6h-u3jph8opje-kubeconfig
-rw-r--r-- 1 ribx ribx 5561 2020-08-07 09:20 /tmp/1kxkfhsdfp9-t03df1vmgd7-kubeconfig
-rw-r--r-- 1 ribx ribx 5561 2020-08-06 15:01 /tmp/1onwpdpxwv8-tcacdzu8f4p-kubeconfig
-rw-r--r-- 1 ribx ribx 5561 2020-08-06 14:01 /tmp/1xgtb16uhcs-zy5oc2ends-kubeconfig
```

**Environment (please complete the following information):**
- Lens Version: 3.5.2 (latest)
- OS: Gentoo Linux
- Installation method (e.g. snap or AppImage in Linux): AppImage

**Comment by nevalla** [comment]
Thanks for reporting this. Kubeconfig handling will change in `3.6.0` and we need to address also this if this is still an issue there.

**Comment by nevalla** [comment]
Fixed in `master`. Lens does not use temporary kubeconfigs anymore. Only Kubeconfig file Lens is creating is when pasting Kubeconfig on cluster add. This Kubeconfig file is cleaned up when cluster is removed from the Lens app. However, I can see there's room for improvements by setting correct permissions to the file.

**Comment by nevalla** [comment]
File permission issue is fixed in #805.
